config/image-relocation-webhook/200-rbac.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ir-sa
+  namespace: image-relocation
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ir-cluster-role
+rules:
+- apiGroups: ["*"]
```

Comment:
can we lock this down to the exact permissions needed?

Reply:
Sure, but I'll need to investigate.

Reply:
Added a `FIXME` comment for now.